nadabot/serializers.py

```
@@ -0,0 +1,15 @@
+from rest_framework.serializers import ModelSerializer
+
+from .models import NadabotRegistry, Provider, Stamp
```

Ruff (0.8.2)

`.models.Stamp` imported but unused

Remove unused import: `.models.Stamp`

(F401)